great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -164,12 +164,12 @@
 T = TypeVar("T", dict, list, str)
 
 
+@public_api
 class AbstractDataContext(ConfigPeer, ABC):
-    """
-    Base class for all DataContexts that contain all context-agnostic data context operations.
+    """Base class for all Data Contexts that contains shared functionality.
```

Comment:
Wanting to check so that I can use the term correctly: `Data Context` is the correct spelling now?

Reply:
Yep! If you are referring to the concept of a Data Context then the space is appropriate. If you are referring to the class then DataContext is appropriate. But now we don't have a DataContext class, so you would use FileDataContext, CloudDataContext, EphemeralDataContext.